etc/nginx/locations/http-available/wazo-router-confd

```diff
@@ -0,0 +1,8 @@
+location ^~ /api/router_confd/ {
+    proxy_pass http://127.0.0.1:8000/;
+    # client_max_body_size 16m;
+
+    proxy_set_header    Host                $http_host;
+    proxy_set_header    X-Script-Name       /api/router_confd;
+    proxy_set_header    X-Forwarded-For     $proxy_add_x_forwarded_for;
+}
```

Comment:
Change the name of the directory to use https instead http.

Reply:
oups

Reply:
For https, i wanted for test to keep it without ssl

Reply:
Why? I'm not sure to understand. We don't expose any API by nginx in http. There is no reason to expose it in http only.